packages/p2p/src/components/my-ads/my-ads-table.jsx

```diff
@@ -26,7 +28,37 @@ const getHeaders = offered_currency => [
 const MyAdsTable = () => {
     const { floating_rate_store, general_store, my_ads_store } = useStores();
     const [selected_advert, setSelectedAdvert] = React.useState(undefined);
-    const local_currency = general_store.client.local_currency_config.currency;
+
+    const generateHintBoxMessage = () => {
+        if (floating_rate_store.rate_type === ad_type.FLOAT) {
+            return floating_rate_store.reached_target_date ? (
+                <Localize i18n_default_text='Your ads with fixed rates have been deactivated. Set floating rates to reactivate them.' />
+            ) : (
+                <Localize
+                    i18n_default_text={
+                        'Floating rates are enabled for {{local_currency}}. Ads with fixed rates will be deactivated. Switch to floating rates by {{end_date}}'
+                    }
+                    values={{
+                        local_currency: general_store.client.local_currency_config.currency || '',
+                        end_date: floating_rate_store.fixed_rate_adverts_end_date || '',
+                    }}
+                />
+            );
+        }
+        return floating_rate_store.reached_target_date ? (
+            <Localize i18n_default_text='Your ads with floating rates have been deactivated. Set fixed rates to reactivate them.' />
+        ) : (
+            <Localize
+                i18n_default_text={
+                    'Fixed rates are enabled for {{local_currency}}. Ads with floating rates will be deactivated. Switch to fixed rates by {{end_date}}'
```

Comment:
Are these correct messages? This goes **against** the idea of asking the user to **NOT** use fixed rates and **set floating rates instead**.

Reply:
There is a use case where a country can be configured to create ads from floating rates to fixed rates

Reply:
But the condition is checking `floating_rate_store.reached_target_date`

Reply:
There are 3 condition. In **line 98**, I am checking if there is a need to change the ads. If that is set as true, then _line 31_, I am checking what is the ad type that is configured to the country. Then based on whether the deadline is past or future, I am displaying the message